### ISSUE TITLE
Build and test failures with latest Android toolchain

### ISSUE THREAD

**Issue by cnorthrop** [KhronosGroup/Vulkan-LoaderAndValidationLayers#1551]
The latest releases of the Android NDK (r14) and SDK Tools (25.3.1) are causing problems with our builds and tests.

If you've already upgraded, please roll back until we get them straightened out.  I'll update this issue when things are working again with latest tools, either through source changes or tool updates.

Links to working versions below:

Android SDK Tools:
Windows - https://dl.google.com/android/repository/tools_r25.2.3-windows.zip
Linux - https://dl.google.com/android/repository/tools_r25.2.5-linux.zip
OSX - https://dl.google.com/android/repository/tools_r25.2.3-macosx.zip

Android NDK (13b known to work):
http://developer.android.com/ndk/downloads/older_releases.html

**Comment by cnorthrop** [comment]
PR #1562 works around the NDK problem, so Android NDK r14 should work now.

**Comment by cnorthrop** [comment]
PR #1673 fixes the packaging problem for Linux and OSX.  Still need to update Windows scripts.

**Comment by mark-lunarg** [comment]
@cnorthrop, is this still an issue?

**Comment by cnorthrop** [comment]
Yes, still an issue for Windows.  Probably will remain an issue until we get everything migrated over to python scripts.
